### ISSUE TITLE
Need Vite support.

### ISSUE THREAD

**Issue by ponikar** [ealush/emoji-picker-react#202]
I am using vite for tooling my web application. When I installed this library in my existing project, It didn't seem to be worked. I have received errors like "global is not found".

If anyone has used this library with Vite, Please help me.

**Comment by mher-s** [comment]
I am using Vite too and I have the same problem there is no support for it.

**Comment by ponikar** [comment]
Is there any way to use this library with vite?

**Comment by ealush** [comment]
At the moment I don't have the capacity to research this issue, but I am
more than willing to accept incoming PRs. Feel free to take a look and
submit the PR, otherwise I am afraid it might take some time.

**Comment by Spencer17x** [comment]
You can set '<script>window.global = window;</script>' solve

**Comment by mher-s** [comment]
@Spencer17x Thanks a lot it works now.

**Comment by ealush** [comment]
Would you like adding it to the README?

**Comment by Spencer17x** [comment]
Of course, but I think we still have to fix this problem, because other libraries won't have this problem. I can pr go to the readme first, and then try pr to fix this problem, what do you think. @ealush

**Comment by Spencer17x** [comment]
I found a way to adapt the vite project without modifying the emoji-picker-react source code, and added the solution to the readme through pr. @ealush

**Comment by federi95** [comment]
Or just edit `vite.config.ts` like this
`export default defineConfig({
  define: {
    global: 'window',
  },
})`
